docs/pelion_ready/device-management.md

```diff
@@ -17,30 +17,30 @@ This guide builds on the [quick connect guide](https://os.mbed.com/guides/connec
 - [Validation and testing](../mbed-os-pelion/device-management-test.html).
 - [Troubleshooting](../mbed-os-pelion/device-management-test.html#troubleshooting).
 
-<span class="tips">**Tip**: To learn more about Device Management Client, please see our [Device Management documentation](https://cloud.mbed.com/docs/latest/client-api-references/index.html).</span>
+<span class="tips">**Tip**: To learn more about Device Management Client, see the [API documentation](https://www.pelion.com/docs/device-management/current/client-api-references/index.html) and the [tutorials](https://www.pelion.com/docs/device-management/current/connecting/device-management-client-tutorials.html).</span>
 
 ## Device Management for your Mbed OS application
 
 ### Requirements
 
-Not every device (microcontroller, module or board) is capable of running device management features. Although you can add or extend some hardware capabilities, such as connectivity, storage and TRNG, others are impossible or inconvenient to extend (for example, RAM or flash).
+Not every device (microcontroller, module or board) is capable of running device management features. Although you can add or extend some hardware capabilities, such as connectivity, storage and TRNG. Others are impossible or inconvenient to extend (for example, RAM or flash).
 
 The minimum requirements to add device management feature to your application are:
 
-- RAM: 128K or more.
-- Flash: 512K or more.
+- RAM: 128 K or more.
+- Flash: 512 K or more.
 - Real Time Clock (RTC).
 - (Optional but recommended) True Random Number Generator (TRNG).
 - A storage device: SD card, SPI flash, QSPI flash or data flash.
 - IP connectivity: Ethernet, Wi-Fi, cellular, 6LoWPAN or Thread.
 
-Additionally, we recommend the latest version of Mbed OS support the device and any additional complementary hardware components, or that they have support for the APIs provided in the latest releases of Mbed OS.
+Additionally, we recommend that the latest version of Mbed OS supports the device and any additional complementary hardware components, or that they have support for the APIs provided in the latest releases of Mbed OS.
```

Comment:
This "that" is unnecessary.
Query: How formal are we when it comes to the subjunctive mood? Do we keep it as "support," or do we relax it?